processor/transaction/_meta/fields.yml

```diff
@@ -25,15 +25,12 @@
             - name: keyword
               type: keyword
           description: >
-            Name of the transaction. This is usually a generic path to the controller/endpoint that handled the
-            transaction.
-
-            TODO: investigate use of "path_hierarchy" tokenizer
+            Generic designation of a transaction in the scope of a single service (eg. 'GET /users/:id').
 
         - name: type
           type: keyword
           description: >
-            The transaction type, e.g. "request", or "job".
+            Keyword of specific relevance in the service's domain (eg. 'request', 'cache', etc)
```

Comment:
I don't see how `cache` is applicable here. `backgroundjob` or `job` sounds more likely. `cache` is applicable in `span.type`.